### PR TITLE
E2E test for multi app run k8s

### PR DESCRIPTION
**Pull request by mukundansundar**
# Description

To be merged after #1333 

Example run https://github.com/dapr/cli/actions/runs/6068518369/job/16461582317#step:14:1368 

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: pkg/kubernetes/run.go [hunk 1]
```diff
@@ -360,7 +361,11 @@ func writeYamlFile(app runfileconfig.App, svc serviceConfig, dep deploymentConfi
 }
 
 func deployYamlToK8s(yamlToDeployPath string) error {
-	_, err := utils.RunCmdAndWait("kubectl", "apply", "-f", yamlToDeployPath)
+	_, err := os.Stat(yamlToDeployPath)
+	if os.IsNotExist(err) {
+		return fmt.Errorf("error given file %q does not exist", yamlToDeployPath)
```
Comment (pravinpushkar) [inline pkg/kubernetes/run.go:366]:
```suggestion
		return fmt.Errorf("error in deploying file %q to k8s environment: %w", yamlToDeployPath, err)
```

File: pkg/kubernetes/run.go [hunk 2]
```diff
@@ -369,9 +374,13 @@ func deployYamlToK8s(yamlToDeployPath string) error {
 
 func deleteYamlK8s(yamlToDeletePath string) error {
 	print.InfoStatusEvent(os.Stdout, "Deleting %q from Kubernetes", yamlToDeletePath)
-	_, err := utils.RunCmdAndWait("kubectl", "delete", "-f", yamlToDeletePath)
+	_, err := os.Stat(yamlToDeletePath)
+	if os.IsNotExist(err) {
+		return fmt.Errorf("error given file %q does not exist", yamlToDeletePath)
```
Comment (pravinpushkar) [inline pkg/kubernetes/run.go:379]:
```suggestion
		return fmt.Errorf("error in getting file info for file %q: %w ", yamlToDeletePath, err)
```

File: pkg/kubernetes/run.go (hunk 1 above)
Comment (mukundansundar) [reply pkg/kubernetes/run.go:366]: this is specifically a does not exist error ... So I think should be fine.

File: pkg/kubernetes/run.go (hunk 2 above)
Comment (mukundansundar) [reply pkg/kubernetes/run.go:379]: same as above.

**Review by pravinpushkar — APPROVED** [review]
lgtm

**Pull request merged**